### DISCUSSION TITLE
Show process progress

### DISCUSSION THREAD

**Discussion by YamilBrachoIn** [vlang/v#15122] [General]
Hi, I am a process that retrieves information from some web pages. I have a for and then a print to show a message, like this

for { 
    print('\r$count lines processed... ')
    ....
   count++
}

However the message is not written to the screen. It only get written when I added a new line or a println(' ') and of course it does not get shown in the same line...Is ether a way to flush the terminal or something to show a message like the way I want to do ?

TIA,
Yamil

**Comment by JalonSolov** [comment]
You can do a lot of things with the terminal using the `term` module:  https://modules.vlang.io/term.html

You can even do UI type things using `term.ui`: https://modules.vlang.io/term.ui.html

**Comment by YamilBrachoIn** [comment]
I already adjust my code using term module

Thanks a lot!!!
Yamil
